### PR TITLE
Add align-section option

### PR DESCRIPTION
**Pull request by Gillou68310**

### CODE REVIEW

File: gas/as.c [hunk 1]
```diff
@@ -1130,6 +1135,42 @@ This program has absolutely no warranty.\n"));
 
 	case OPTION_HASH_TABLE_SIZE:
 	  break;
+
+	case OPTION_ALIGN_SECTION:
+    int i;
+    long val = 0;
+    char *res = NULL;
+    char *align = strchr(optarg, ':');
```
Comment (AngheloAlf) [inline gas/as.c:1143]: formatting looks weird, we'll probably want to do something about it later, but no big deal right now

Comment (Gillou68310) [reply gas/as.c:1143]: Yeah the whole shit is mixing tab/space formating hell!

Comment (AngheloAlf) [reply gas/as.c:1143]: You could take the vscode files from another repo to get proper indentation
https://github.com/decompals/mips-binutils-egcs-2.9.5/tree/master/.vscode

Comment (Gillou68310) [reply gas/as.c:1143]: Sure I was just lazy to reformat the whole src

**Pull request merged**